src/autogluon_dashboard/dashboard.py

```diff
@@ -2,23 +2,64 @@
 import subprocess
 import sys
 
-if __name__ == "__main__":
-    csv1 = sys.argv[2]
-    csv2 = sys.argv[4]
+import boto3
+import botocore
+
+BUCKET = "dashboard-test-yash"
```

Comment:
Add TODO for changing to dynamically generate bucket and prefix later

Reply:
Okay!